sys/shell/commands/sc_gnrc_netif.c

```diff
@@ -372,6 +422,20 @@ static void _netif_list(kernel_pid_t iface)
     if (res >= 0) {
         printf(" NID: 0x%" PRIx16, u16);
     }
+#ifdef MODULE_LORA
```

Comment:
You removed the other `#ifdef`s. Why not this one (which except for size optimization is the least useful as it wouldn't show anything if compiled in but the options were not supported)?